flash/core/data/utilities/paths.py

```diff
@@ -133,6 +133,18 @@ def list_valid_files(
     return [path for path in paths if has_file_allowed_extension(path, valid_extensions)]
 
 
+def _show_filter_valid_files_message(
+    invalid_extensions: Optional[Tuple[str, ...]] = None, valid_extensions: Optional[Tuple[str, ...]] = None
```

Comment:
Thanks for being verbose with the typing annotations here. Do you want to make them optional? If they are ever `None` (optional), the statement: `", ".join(...)` will fail.

From what I see, we should (/will) always have `valid_extensions`, but `invalid_extensions` can be an empty list.

Reply:
@JustinGoheen - Yep, that's great. What I meant was, the default values in this function for `invalid_extensions` and `valid_extensions` are `None`, for which the function will fail (as we do: `", ".join(valid_extensions)`.

I know we only use it once, so it's all good for now. But for completeness, we can skip the warning if any of them is `None`?

Reply:
it looks like the code at ln 156 handles the return if valid_extensions is None, right?

Reply:
Now it's all good as this function seems to be moved to the filter_valid_files function. :) Thanks!